src/playback/summary.jsx

```diff
@@ -226,16 +252,22 @@ class PlaybackSummary extends React.Component {
                       item
                       xs={1}
                       style={{ position: 'relative', padding: '0.1rem' }}>
-                      <div className={classes[score]}>
-                        <div
-                          style={{
-                            position: 'absolute',
-                            top: '50%',
-                            left: '50%',
-                            transform: 'translate(-50%, -50%)',
-                          }}>
+                      <div
+                        className={classes[score]}
+                        style={{
+                          display: 'flex',
+                          alignItems: 'center',
+                          justifyContent: 'center',
+                        }}>
+                        <Tooltip
+                          content={this.tooltipContent(
+                            score,
+                            encoding,
+                            platform.id,
+                            id
+                          )}>
                           {label}
-                        </div>
+                        </Tooltip>
```

Comment:
move tooltip out so that it shows when hovering over the colored square, not just the text in the square.

Reply:
can the tooltip be wider?